### PR TITLE
D8CORE-355 Use decanter breakpoints in step definition

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Created step definition for easily switching breakpoints from decanter.

# Needed By (Date)
- End of sprint (8/30)

# Urgency
- low

# Steps to Test
1. checkout https://github.com/SU-SWS/acsf-cardinalsites/pull/16
1. delete `tests/behat/local.yml` file
1. `composer install --prefer-source`
1. `cd vendor/su-sws/stanford-caravan`
1. `git checkout D8CORE-355-decanter-breakpoints`
1. modify a behat to use the format `Then I set the window to "medium"`
1. run the behat test and ensure it doesnt error on that step.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 1]
```diff
@@ -30,6 +30,31 @@ public function beforeScenario(BeforeScenarioScope $scope) {
...
+   *
+   * @param string $size
+   *   Small, Medium, Large, or 'Extra Large' screen.
+   *
+   * @Then I set the window size to :size
+   */
+  public function iSetWindowSize($size) {
+    $sizes = [
+      'small' => [576, 320],
+      'medium' => [768, 1024],
```
Comment (sherakama) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:46]: Are these `max`, `min`?
```suggestion
      'medium' => [1024, 768],
```

Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:46]: window width, height values. i just chose some height that would simulate a portrait tablet.

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 2]
```diff
@@ -30,6 +30,31 @@ public function beforeScenario(BeforeScenarioScope $scope) {
...
+  /**
+   * Set the window to a breakpoint size.
+   *
+   * @param string $size
+   *   Small, Medium, Large, or 'Extra Large' screen.
+   *
+   * @Then I set the window size to :size
+   */
+  public function iSetWindowSize($size) {
+    $sizes = [
```
Comment (pookmish) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]:
```suggestion
    // Sizes are height, width values for the window size.
    $sizes = [
```

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 3]
```diff
@@ -30,6 +30,32 @@ public function beforeScenario(BeforeScenarioScope $scope) {
...
+   * Set the window to a breakpoint size.
+   *
+   * @param string $size
+   *   Small, Medium, Large, or 'Extra Large' screen.
+   *
+   * @Then I set the window size to :size
+   */
+  public function iSetWindowSize($size) {
+    // Sizes are height, width values for the window size.
+    $sizes = [
```
Comment (sherakama) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]:
```suggestion
    $sizes = [
      'extra small' => [575, 320],
```

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 4]
```diff
@@ -30,6 +30,32 @@ public function beforeScenario(BeforeScenarioScope $scope) {
...
+  /**
+   * Set the window to a breakpoint size.
+   *
+   * @param string $size
+   *   Small, Medium, Large, or 'Extra Large' screen.
+   *
+   * @Then I set the window size to :size
+   */
+  public function iSetWindowSize($size) {
+    // Sizes are height, width values for the window size.
```
Comment (sherakama) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]:
```suggestion
    // Sizes are width, height values for the window size.
```

Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]: doh, yeah they are

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php (hunk 3 above)
Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]: 1px different than `small`?

Comment (sherakama) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]: Yeah, there is a breakpoint that is from 0 - 575 then small starts at 576 and goes up.

Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]: :+1:

Comment (sherakama) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]: `xs` => 0 - 575
`sm` => 576 - 767
`md` => 768 - 991
`lg` => 992 - 1199
`xl` => 1200 - 1499
`2xl` => 1500+

Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php:43]: `sm => 576 - 767` right?

**Review by sherakama — APPROVED** [review]

**Pull request merged**